Calendar/Get-CalendarDiagnosticObjectsSummary.ps1

```diff
@@ -596,17 +603,20 @@ function SetIsIgnorable {
         $CalLog
     )
 
-    if ($ShortClientName -like "EBA*" `
+    if ($ShortClientName -like "TBA*SharingSyncAssistant" `
```

Comment:
Let's avoid using back ticks here for a new line. Instead just place your `-or` or `-and` on the previous line and PowerShell knows to continue to the next line.